pkg/ipmi/ioctl.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
can you call this ioctl_unix.go

might as well keep the naming conventions for future work.

Reply:
I think this is not linux specific, I think it should work on BSD as well:
https://github.com/freebsd/freebsd/blob/master/sys/dev/tdfx/tdfx_io.h#L54

I have't tried it though.

Reply:
unix suffix usually includes both linux and *bsd.

name it `ioctl_unix.go` and add a constraint like

```go
// +build aix darwin dragonfly freebsd js,wasm linux netbsd openbsd solaris
```

or whatever subset you think it'll work for.

Reply:
I misread `_linux.go`. I'll rename it.